src/pages/playground.svelte

```diff
@@ -0,0 +1,89 @@
+<script>
+  import Editor from '../components/CodeMirror.svelte';
+  import type CodeMirror from 'codemirror';
+  import Visualizer from '../components/Visualizer.svelte';
+  import { shaderTexts } from '../modules/utils';
+
+  const fragCallback = (cm: CodeMirror.Editor) => {
```

Comment:
If you're gonna try to load up one file, it should probably be this one. Pages in `src/pages/` correspond one-to-one with routes, so since this is `/playground`, you can just go to `/playground` to see it.